.travis.yml

```diff
@@ -10,7 +10,8 @@ before_install:
   - npm install -g grunt@0.4.5
   - npm install -g grunt-cli
 install: npm install
-scripts:
+script:
+  - grunt
```

Comment:
That makes sense and would also mean that we can remove `grunt` from travis as I think that `test` will be run by default. I'll try that out.